README.md

```diff
@@ -273,14 +273,6 @@ to interpret Carbon code and print its output. You can try it out immediately at
 To build the Carbon explorer yourself, follow these instructions:
 
```

Comment:
Since we're removing the dependencies from the instructions, maybe frontload the fact that they need to be set up?

```suggestion
To build the Carbon explorer yourself, you'll need to get some dependencies setup (Bazel, Clang, libc++), and then you can follow these instructions:

```

And then we can tweak the wording below...